packages/jest-haste-map/src/index.ts

```diff
@@ -113,6 +113,7 @@ const CHANGE_INTERVAL = 30;
 const MAX_WAIT_TIME = 240000;
 const NODE_MODULES = path.sep + 'node_modules' + path.sep;
 const PACKAGE_JSON = path.sep + 'package.json';
+const VCS_DIRECTORIES = ['.git', '.hg'].map(vcs => '/' + vcs + '/').join('|');
```

Comment:
why has this lost `path.sep`?

Reply:
I have reverted changed to the version which I knew was passing